cluster/addons/fluentd-elasticsearch/es-image/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
commit history suggests this was set to `/bin/sh` in order to "rebase onto alpine", however that's not actually what the image is `FROM` today 🤷‍♂ 

```
$ docker run --rm --entrypoint=/bin/sh gcr.io/fluentd-elasticsearch/elasticsearch:v6.7.2 -c 'cat /etc/os-release'
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```